### PR TITLE
Adds the ability to soak rags and cotton balls on disinfectant. 

### PR DESCRIPTION
**Pull request by Zhid13**
Due to the rather harsh avaliability of disinfectant to early game survivors, this patch lets you soak 4 rags with one charge of disinfectant, with a 20% less probabilities to cure the infected wound. It adds the same to cotton balls, with only 2 of them being soaked.

### CODE REVIEW

**Comment by Zhid13** [comment]
Corrected it.

**Comment by smolbird** [comment]
Soaking rags and cotton balls make them suddenly unable to stop bleeding?

**Comment by Zhid13** [comment]
Yes, because you are supposed to rub them against the wound to apply the disinfectant instead of trying to stop the bleeding

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -1288,5 +1288,54 @@
         "addiction_potential" : 40,
         "addiction_type" : "sleeping pill",
         "use_action" : "XANAX"
+    },
+    {
+        "id" : "disinrag",
+        "type" : "COMESTIBLE",
```
Comment (DracoGriffin) [inline data/json/items/comestibles/med.json]: Suggest adding `"copy-from":` to condense as well as any fallbacks if the original items are altered.

File: data/json/items/comestibles/med.json [hunk 2]
```diff
@@ -1288,5 +1288,54 @@
...
+            "limb_power" : 10,
+            "head_power" : 10,
+            "torso_power" : 10,
+            "bite" : 0.75,
+            "move_cost" : 100
+        }
+    },
+    {
+        "id" : "disincotton_ball",
+        "type" : "COMESTIBLE",
```
Comment (DracoGriffin) [inline data/json/items/comestibles/med.json]: Suggest same `"copy-from":` line here from the base item (in this case, the non-soaked cotton balls).

**Pull request closed without merging**